packages/flutter_tools/lib/src/build_runner/devfs_web.dart

```diff
@@ -837,7 +837,8 @@ class ReleaseAssetServer {
     } else {
       for (final Uri uri in _searchPaths) {
         final Uri potential = uri.resolve(request.url.path);
-        if (potential == null || !globals.fs.isFileSync(potential.toFilePath())) {
+        if (potential == null || !globals.fs.isFileSync(
+          potential.toFilePath(windows: globals.platform.isWindows))) {
```

Comment:
missing windows check, was creating this as a windows style file path even if configured paths were possix style